src/ccl_emu17.c

```diff
@@ -138,21 +138,30 @@ static void emuInit() {
 } // emuInit()
 
 // Actual emulation
-void ccl_pkemu(double *xstar, double **ystar, int* status, ccl_cosmology* cosmo) {
-    
+void ccl_pkemu(double *xstar, int sizeofystar, double *ystar, int* status, ccl_cosmology* cosmo)
+{
```

Comment:
Why the interface change here?

Reply:
Before this routine set `*ystar` to `NULL` to mark when it failed, which could create memory leaks. This is now handled through the `status` flag, and I pass the size of `ystar` to add an extra check that it has been correctly allocated.